datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/dep.py

```diff
@@ -115,6 +116,26 @@ def verify_dependency(source, name, python_versions, file):
             requirement = Requirement(dependency_definition)
             specifier_set = requirement.specifier
 
+            # git support: https://pip.pypa.io/en/stable/topics/vcs-support/
+            pip_vcs_support_link = "https://pip.pypa.io/en/stable/topics/vcs-support/"
+            valid_schemes = ["git+file", "git+https", "git+ssh", "git+http", "git+git", "git"]
+
+            if requirement.url:
+                u = urlparse(requirement.url)
+                if u.scheme not in valid_schemes:
+                    message = f'Invalid URL scheme found for dependency `{name}`: {format_check_usage(checks, source)}. Please see `{pip_vcs_support_link}` for more information.'
```

Comment:
hmm, what if our list of valid schemes becomes out of date with the docs? I think I'd be confused :)

Let's print the list of what we consider as valid in the error